hack/test.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
-go vet $(go list ./... | grep -v '/vendor/')
-go test $(go list ./... | grep -v '/vendor/')
+godep go vet $(go list ./... | grep -v '/vendor/')
+godep go test $(go list ./... | grep -v '/vendor/')
 
-[ -d ./bin ] && mkdir bin
+[ -d ./bin ] || mkdir bin
 
 godep go build -o bin/workflow
 
 for file in ./pkg/*
 do
-    go test -coverprofile=/tmp/cover.out $file
-    go tool cover -func=/tmp/cover.out
+    CFILE=/tmp/cover.out
+    godep go test -coverprofile=/tmp/cover.out $file
```

Comment:
Could give every cover file a distinct name, optionally generate. This solves the problem of displaying results twice if no test file is present and will allow coverage services to generate a report + badge.